src/main/java/com/fundation/webservice/model/CriteriaVideo.java

```diff
@@ -0,0 +1,90 @@
+package com.fundation.webservice.model;
+
```

Comment:
Please add a custom description of this class.

Reply:
solved